apps/condo/domains/ticket/utils/testSchema/index.js

```diff
@@ -166,12 +166,21 @@ async function makeClientWithTicket () {
 }
 
 module.exports = {
-    Ticket, createTestTicket, updateTestTicket, ticketStatusByType,
-    TicketStatus, createTestTicketStatus, updateTestTicketStatus,
-    TicketFile, createTestTicketFile, updateTestTicketFile,
-    TicketChange, createTestTicketChange, updateTestTicketChange,
+    Ticket,
+    TicketFile,
+    TicketChange,
+    TicketStatus,
     TicketSource,
     TicketClassifier,
+    createTestTicket,
+    updateTestTicket,
+    ticketStatusByType,
+    createTestTicketStatus,
+    updateTestTicketStatus,
+    createTestTicketFile,
+    updateTestTicketFile,
+    createTestTicketChange,
+    updateTestTicketChange,
```

Comment:
Probably, it's better to change the file template [here](https://github.com/open-condo-software/condo/blob/master/packages/%40core.codegen/templates/schema00/domains/%7B%7Bdomain%7D%7D/utils/testSchema/index.js.patch#L49-L51) if you want to change the export style